### ISSUE TITLE
Feature Request: Visual Tagging of Tree Nodes (Icon and/or Text Foreground Color)

### ISSUE THREAD

**Issue by gary17** [giuspen/cherrytree#115]
It is often very useful to be able to tag ("differentiate") a particular tree node in some way, in order to emphasize the underlying tree structure (the most important work at level 1, red, subtasks at level 2, yellow, etc.) or in order to mark a particular tree node as special (important, work-in-progress, etc.)

This kind of visual tagging could be accomplished through a setting of an icon and/or a text foreground color in the tree "Node Properties" dialog box.

**Comment by giuspen** [comment]
If you bookmark the node, a red pin will appear beside the node in the tree

**Comment by gary17** [comment]
Good point, but when you have, say, 50+ tree nodes, it becomes critical to be able to use MULTIPLE different colors and/or icons representing different levels, status indicators, etc. Kind of like Thunderbird at-a-glance color tagging for Important, Work, Personal, To Do, Later...

**Comment by giuspen** [comment]
I see your point, maybe a custom coloured square beside the node icon or in
place of the node icon could fit your needs

**Comment by gary17** [comment]
Thunderbird "color tagging" will change the foreground text color for an entire e-mail "row" when not selected (inactive) and the background color when selected (active). I think the text color for an entire tree node would be more effective (differentiate "stronger") than any kind of an icon alone. Also, multiple icons on a single tree node (a tag icon, a cherry/bullet icon and a bookmark icon) might be confusing to new users as to what they all mean. I would "vote" either a text color on its own or a text color plus an option to select any external icon as a tree node icon (instead of a cherry/bullet). Might be just me, maybe other users will have more feedback on this.
